app/src/app/api/v0/assistants/route.ts

```diff
@@ -0,0 +1,31 @@
+import { apiHandler } from "@/util/api";
+import { openai } from "@/util/openai";
+import { NextResponse } from "next/server";
+
+// Create a new assistant
+export const POST = apiHandler(async () => {
+  // Currently we create the assistant via OpenAI web interface and do not implement this function here
+  return NextResponse.json({ error: "Method Not Allowed", status: 405 });
+  // TODO: should be an admin function
+  const userID = "id_123456"; // TODO: Get userID
+
+  const systemPrompt = `Your name is CLIMA and you are a climate assistant for creating 
+'Global Protocol for Community-Scale (GPC) Greenhouse Gas (GHG) Inventories' using CityCatalyst, 
+an open source tool for creating climate inventories by Open Earth Foundation. 
+You try to be as helpful as possible when answering the user\'s questions about their inventory 
+or any climate science or data science related questions. 
+Try to be as scientific as possible. Use primarily the provided context of the attached documents,
+or context provided by the user.`;
+
+  const assistant = await openai.beta.assistants.create({
+    instructions: systemPrompt,
+    name: `ClimateAdvisor_v0.1_${userID}`,
+    model: "gpt-3.5-turbo", // gpt4o-mini not working currently
+    temperature: 0.2,
+    tools: [{ type: "file_search" }],
+    tool_resources: {
+      file_search: { vector_store_ids: ["vs_oS3uJ02f4enB7oK5pSU8pIHq"] }, // TODO: Hardcoded vectorstore ID
```

Comment:
Or can this be acquired from the OpenAI API somewhere, so we list the available vector stores for the given assistant? This way it can be used more easily by self-hosting users.

Reply:
Yes absolutely. This should be either set as an env variable or created by an admin user. 

For OEF hosted CC, it makes sense that we provide this out of the box and the users do not have to do anything (like getting API key, creating assistant and vector stores,....) 
For 3rd party hosted systems I think this is something we also have to discuss with product. Since those parties will use their own API keys, they also need to create their own assistant, vector stores, .... on the future admin page. 

So as long as we do not have this admin page, I will take the vector store_id out. It is something that we will still have to implement then.

Reply:
I created a ticket for this, to define the user story and further implementation